sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -1267,6 +1282,55 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
     return studyBreaksAndWeeks;
   }
 
+  /**
+   * Adjust the study breaks when there is a change in study end date
+   * during scholastic standing.
+   * * This method is used when there is a change in study end date, which,
+   * * will effect the study break period, which will not respect the basic
```

Comment:
I believe that it should be "will affect" instead "will effect", does it make sense?